src/Routes/SeasonsController.ts

```diff
@@ -2,8 +2,37 @@ import * as express from "express";
 import { IDataProvider } from "../DataProviders/IDataProvider";
 import { BaseController } from "./BaseController";
 
-/*
- * Seasons controller
+/**
+ * @swagger
+ *
+ * definitions:
+ *   Seasons:
```

Comment:
Season[] or ISeasonInfo[]

Reply:
Array specified in the schema reference.